### PR TITLE
RPP Audio Support HIP - Downmixing

### PR DESCRIPTION
**Pull request by r-abishek**
- Adds RPP audio support for Downmixing on HIP backend.
- Adds relevant qa tests and performance tests.

### CODE REVIEW

File: src/modules/hip/kernel/down_mixing.hpp [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    int srcLength = srcDimsTensor[id_z].x;
+    int channels = srcDimsTensor[id_z].y;
+
+    if (id_x >= srcLength)
+        return;
+
+    float outVal = 0.0f;
+    uint srcIdx = id_z * srcStride + id_x * channels;
+    int i = 0;
+    int alignedChannels = (channels / 8) * 8;
```
Comment (rrawther) [inline src/modules/hip/kernel/down_mixing.hpp:22]: This is rounding down. Is that what you want?

**Review by rrawther — APPROVED** [review]
Added some comments

File: src/modules/hip/kernel/down_mixing.hpp (hunk 1 above)
Comment (sampath1117) [reply src/modules/hip/kernel/down_mixing.hpp:22]: Yes Rajy
Since we want to do 8 pixel vectorized processing for alignedChannels value

**Comment by kiritigowda** [comment]
@r-abishek - conflicts to be resolved

**Comment by r-abishek** [comment]
@kiritigowda resolved

**Comment by kiritigowda** [comment]
@r-abishek please resolve merge conflicts.

**Comment by r-abishek** [comment]
@kiritigowda resolved, this could be merged too

**Pull request merged**